src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -45,14 +45,14 @@
 						:aria-label="t('spreed', 'Share files to the conversation')"
 						:aria-haspopup="true">
 						<ActionButton
-							v-if="canShareAndUploadFiles"
+							v-if="canUploadFiles"
```

Comment:
Should the Actions have `v-if="canUploadFiles && canShareFiles"` ?

Reply:
the canUploadFiles computed attr is already based on the same conditions like canShareFiles, so not needed

Reply:
as discussed, I've now added an extra v-if on the div that contains Actions to avoid rendering it empty